pkg/koordlet/resourceexecutor/executor.go

```diff
@@ -122,6 +123,9 @@ func (e *ResourceUpdateExecutorImpl) LeveledUpdateBatch(cacheable bool, updaters
 			if err != nil && sysutil.IsResourceUnsupportedErr(err) {
 				klog.V(5).Infof("failed merge update unsupported resource %s, err: %v", updater.Key(), err)
 				continue
+			} else if err != nil && os.IsNotExist(err) {
```

Comment:
need to identify the different cases that return a not-exist error. we can ignore (i.e. log more verbosely) only some of the cases:

1. the cgroup (directory) does not exist. possibly the container is restarting
2. the cgroup exists but the target cgroup file is missing. possibly the cgroup file is unsupported by the system or the cgroup feature is disabled
3. ...